core/src/main/java/org/radargun/stages/monitor/MonitorStartStage.java

```diff
@@ -75,6 +68,11 @@ private void addMonitors(AbstractMonitors monitors, Timeline timeline) {
       monitors.addMonitor(new MemoryUsageMonitor(jmxConnectionProvider, timeline));
       monitors.addMonitor(new GcMonitor(jmxConnectionProvider, timeline));
       monitors.addMonitor(new OpenFilesMonitor(jmxConnectionProvider, timeline));
+      if (SystemUtils.IS_LINUX) {
+         monitors.addMonitor(new RssMonitor(timeline));
+      } else {
+         log.warn("Rss monitor only works on Linux.");
```

Comment:
User did not ask for RSS explicitly, don't bother him with warnings about running wrong OS :)